src/handlers/commands.ts

```diff
@@ -1,65 +1,75 @@
-import { SlashCommandBuilder, SlashCommandSubcommandGroupBuilder, Guild, type Client } from "discord.js";
-import { pathToFileURL } from "url";
-import { join } from "path";
+import {
+  Guild,
+  SlashCommandBuilder,
+  SlashCommandSubcommandGroupBuilder,
+  type Client
+} from "discord.js";
 import { readdirSync } from "fs";
-import database, { getSettingsTable } from "../utils/database.js";
-import { QuickDB } from "quick.db";
+import { join } from "path";
+import { pathToFileURL } from "url";
+import { getDisabledCommands } from "../utils/database/disabledCommands";
 
-const COMMANDS_PATH = join(process.cwd(), "src", "commands");
-export default class Commands {
+export class Commands {
   client: Client;
   commands: any[] = [];
-  db: QuickDB<any>;
-
-  constructor(client?: Client) {
+  constructor(client: Client) {
     this.client = client;
   }
 
-  // Load the commands into this.commands
-  private async createSubCommand(name: string, ...disabledCommands: string[]): Promise<SlashCommandBuilder> {
+  private async createSubCommand(
+    name: string,
+    ...disabledCommands: string[]
+  ): Promise<SlashCommandBuilder> {
+    const commandsPath = join(process.cwd(), "src", "commands");
     const command = new SlashCommandBuilder()
       .setName(name.toLowerCase())
       .setDescription("This command has no description.");
```

Comment:
Same thing as before, why does it have to be like that ?

Reply:
why does what to be like that?

Reply:
The description being empty

Reply:
because the commands change that, thats a fallback for when a developer doesnt provide a description